example/javalib/linting/3-palantirformat/build.mill

```diff
@@ -0,0 +1,18 @@
+package build
+
+import mill._
+import mill.javalib.palantirformat._
+
+object `package` extends RootModule with PalantirFormatModule {
+}
+
+/** Usage
+
+> ./mill palantirformat                                                 # format all Java source files
+...formatting java sources ...
+
+> ./mill mill.javalib.palantirformat.PalantirFormatModule/ __.sources   # format all Java source files in existing modules
+...formatting java sources ...
+*/
+
+/** See Also: src/A.java */
```

Comment:
Let's move this above the `Usage` block, since it shows the raw source code before the usage commands are run